src/shared-components/progressBar/style.ts

```diff
@@ -3,35 +3,42 @@ import { css, keyframes } from '@emotion/core';
 
 import { COLORS, ANIMATION, PROGRESS_BAR_STATUS } from '../../constants';
 
+import { ProgressBarStatusType } from '.';
+
 const loadingProgression = keyframes`
   from { transform: translateX(-100%); }
   to { transform: translateX(-5%); }
 `;
 
-export const OuterContainer = styled.div`
+export const OuterContainer = styled.div<{
+  backgroundColor: string;
+  height: number;
+  status: ProgressBarStatusType;
+}>`
   position: absolute;
   top: 0;
   left: 0;
   right: 0;
   background: ${props => props.backgroundColor};
   overflow: hidden;
-  transition: opacity ${ANIMATION.defaultIming} ease-in-out 500ms;
+  transition: opacity ${ANIMATION.defaultTiming} ease-in-out 500ms;
   width: 100%;
   z-index: 2;
   height: ${props => `${props.height}px`};
 
   ${({ status }) =>
-    [PROGRESS_BAR_STATUS.success, PROGRESS_BAR_STATUS.error].includes(status) &&
+    (status === PROGRESS_BAR_STATUS.success ||
```

Comment:
Was the change from using the array a TypeScript thing, or personal preference?

Reply:
TS protested that `loading` could never be assigned to the ['success','error'] array I decided to refactor instead of ts-ignoring